packages/bits/spec/components/form-field/form-field.a11y.ts

```diff
@@ -28,73 +28,74 @@ describe("a11y: form-field", () => {
     });
 
     it("button", async () => {
-        const accessibilityScanResults = await axeBuilder(browser.driver).include(`.${ButtonAtom.CSS_CLASS}`).disableRules("color-contrast").analyze();
+        const accessibilityScanResults = await new AxeBuilder(browser.driver).include(`.${ButtonAtom.CSS_CLASS}`).disableRules("color-contrast").analyze();
```

Comment:
can we new this up in a beforeAll to reuse the same instance every time, instead of creating a new one for each test?

Reply:
Unfortunately not. If we'll use a single instance of axe it'll preserve the results of any previously run test, which will result in false negatives.

Creating a new instance is fine according to their docs:
https://github.com/dequelabs/axe-core-npm/tree/develop/packages/webdriverjs